### ISSUE TITLE
Unable to list git ignored files when `IGNORED_GITIGNORED_FILES: true`

### ISSUE THREAD

**Issue by davidB** [nvuillam/mega-linter#608]
**Describe the bug**

When using `IGNORED_GITIGNORED_FILES: true` (error on call to git status --ignored and no files ignored), by fixing the call to external process and by using git ls-files --exclude-standard --ignored --others instead of git status to have an easier output to parse to generate the list of files to ignored.

```
    Unable to list git ignored files (Cmd('g') not found due to: FileNotFoundError('[Errno 2] No such file or directory: 'git status --ignored'')
      cmdline: g i t   s t a t u s   - - i g n o r e d)
```

**To Reproduce**

1. Go to a local project (eg mega-linter) with `.gitignore` files and files ignored by .gitignore (eg: `venv`, `node_modules`, `target`,...)
2. Add a `.mega-linter.yml`

    ```sh
    echo "IGNORE_GITIGNORED_FILES: true" > .mega-linter.yml
    ```

3. run mega-linter on local project

    ```sh
    docker run --rm -it -v /var/run/docker.sock:/var/run/docker.sock:rw -v $PWD:/tmp/lint:rw nvuillam/mega-linter:v4
    ```

  - error is reported
  - files are not ignored (so lot error could be reported by third party,...)

In this sample, run in on a rust project

**Expected behavior**

Ignoring files following the `.gitignores` rules

**Screenshots**

```sh
> docker run --rm -it -v /var/run/docker.sock:/var/run/docker.sock:rw -v $PWD:/tmp/lint:rw nvuillam/mega-linter-rust:v4
...
Listing all files in directory [/tmp/lint], then filter with:
- File extensions: , .R, .RMD, .Rmd, .bash, .c, .c++, .cc, .cdxml, .clj, .cljc, .cljs, .coffee, .cpp, .cs, .css, .cu, .cuh, .cxx, .dart, .dash, .edn, .env, .feature, .go, .gradle, .graphql, .groovy, .gvy, .h, .h++, .hcl, .hh, .hpp, .htm, .html, .hxx, .java, .js, .json, .json5, .jsonc, .ksh, .kt, .kts, .lua, .md, .nf, .p6, .php, .pl, .pl6, .pm, .pm6, .pp, .proto, .ps1, .ps1xml, .psd1, .psm1, .psrc, .pssc, .py, .r, .raku, .rakumod, .rakutest, .rb, .rs, .rst, .saas, .scala, .scss, .sh, .smk, .sql, .t, .tex, .tf, .ts, .vb, .xml, .yaml, .yml
- File names (regex): Dockerfile, Jenkinsfile, Snakefile
Unable to list git ignored files (Cmd('g') not found due to: FileNotFoundError('[Errno 2] No such file or directory: 'git status --ignored'')
  cmdline: g i t   s t a t u s   - - i g n o r e d)
Kept [1492] files on [2806] found files
...
```

**Additional context**

**Comment by davidB** [comment]
I do the flow in the revert order, I first push the PR https://github.com/nvuillam/mega-linter/pull/605

Sorry
